### PR TITLE
Update api spec

### PR DESCRIPTION
**Pull request by yaokanmon**

### CODE REVIEW

File: java/api/openapi.yaml [hunk 1]
```diff
@@ -558,7 +558,7 @@ paths:
         style: form
```
Comment (coleby) [inline java/api/openapi.yaml:558]: Is this file auto generated from the json?

File: ts/src/index.ts [hunk 2]
```diff
@@ -61,5 +63,6 @@ export class KanmonPlatformApi {
     this.offers = new OffersApi(this._configuration);
     this.payments = new PaymentsApi(this._configuration);
     this.prequalifications = new PrequalificationsApi(this._configuration);
+    this.integratedMCAReceivablesApi = new IntegratedMCAReceivablesApi(this._configuration);
```
Comment (coleby) [inline ts/src/index.ts:66]: Can we remove the `Api` suffix off the end of this like the others? That way it follows the convention?

Comment (coleby) [reply ts/src/index.ts:66]: Looking at Java, they all have an Api suffix. I think we should keep this consistent between SDKs. @danielwong2268 which way do you prefer? Be our tie breaker!

Comment (danielwong2268) [reply ts/src/index.ts:66]: yas!

File: java/api/openapi.yaml (hunk 1 above)
Comment (yaokanmon) [reply java/api/openapi.yaml:558]: Yeah, this is auto generated from the JSON. Good catch BTW, will do more digging why it needs to generate this file. I think just the JSON one should be enough. Added to the Java SDK ticket. https://linear.app/kanmon/issue/ENG-1420/create-java-sdk

File: ts/src/index.ts (hunk 2 above)
Comment (yaokanmon) [reply ts/src/index.ts]: Good catch, changed as suggested.

**Review by coleby — APPROVED** [review]
Looks good!

**Pull request merged**